tests/test_statelib_state.py

```
@@ -0,0 +1,268 @@
+    assert state.to_pytree() == tree
+    assert all(isinstance(k, tuple) for k in state.raw_mapping)
+
+
+def test_flat_state_to_pytree_multiple_internal_states_raises() -> None:
+    state1: sl.FlatState[int] = sl.FlatState.from_pytree({"a": 1})
+    state2: sl.FlatState[int] = sl.FlatState.from_pytree({"b": 2})
+    merged: sl.FlatState[int] = sl.merge_states(state1, state2)
+
+    with pytest.raises(
+        ValueError,
+        match=f"Cannot convert to pytree with 2 internal "
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)